src/patternfly/components/Wizard/wizard.scss

```diff
@@ -320,6 +329,12 @@
   line-height: var(--pf-c-wizard__toggle-icon--LineHeight);
 }
 
+.pf-c-wizard__drawer-toggle {
```

Comment:
Could the toggle button get lost if it isn't sticky and is always at the top of the scrolled content?